composer.json

```diff
@@ -15,23 +15,23 @@
         }
     ],
     "require": {
-        "php": "^7.1",
-        "symfony-cmf/routing": "^2.1.0@dev",
-        "symfony/framework-bundle": "^3.4 || ^4.3 || ^5.0",
+        "php": "^7.2",
+        "symfony-cmf/routing": "dev-master",
```

Comment:
note to self: when we see this branch build, lets tag the routing component and then adjust this version